web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -196,6 +196,18 @@ export class LayerDialogComponent implements OnDestroy {
     }
   }
 
+  private isFieldOptionsValid(formFieldEditor: FormFieldEditorComponent) {
+    if (!formFieldEditor.optionEditors) {
+      return;
```

Comment:
Does `undefined` have special meaning? If not this should return true or false.

Reply:
Done